<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkForm</class>
 <widget class="QWidget" name="MarkForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="markNameLabel">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="markName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="responsibleLabel">
       <property name="text">
        <string>Ответственный</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="responsible"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateEditLabel">
       <property name="text">
        <string>Выполнить до</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="importanceLabel">
       <property name="text">
        <string>Важность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="importance">
       <item>
        <property name="text">
         <string>критично</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>важно</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>в рабочем порядке</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>в свободное время</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Статус</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="status">
       <item>
        <property name="text">
         <string>новый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>в работе</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>на проверку</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Категория</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="category">
       <item>
        <property name="text">
         <string>электроснабжение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>водоснабжение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>водоотведение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>отопление</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>вентиляция</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>тепловые сети</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>слаботочные сети</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>газоснабжение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>технологические решения</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>архитектурные решения</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="layerLabel">
       <property name="text">
        <string>Слой</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="layerCB">
       <item>
        <property name="text">
         <string>базовый</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="description"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="loadImage">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:4;
border-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loadImageLabel">
       <property name="text">
        <string>Загрузить изображение</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="galeryLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:4;
border-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toArchieve">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:4;
border-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:4;
border-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>description</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
